src/definitions/collections/table.less

```diff
@@ -641,7 +641,6 @@
         font-weight: @definitionHeaderFontWeight;
         color: @definitionHeaderColor;
         box-shadow: -@coloredBorderSizeCover -@coloredBorderSize 0 @coloredBorderSizeCover @definitionPageBackground;
-        -moz-transform: scale(1);
```

Comment:
introduced in https://github.com/fomantic/Fomantic-UI/pull/884

I tried the fiddle in FF and the fix seems to be not needed anymore, but maybe I do not know all the conditions to reproduce...

Reply:
That fix is unfortunately still needed for latest Firefox, so please revert the remove.
See https://jsfiddle.net/7e0yf5pu/ using latest 2.9.0 and try to remove the attribute via web insepector

![ffdeftablefixstillneeded](https://user-images.githubusercontent.com/18379884/208511076-708d2fa1-bac2-4830-9aba-48c47f885cd1.gif)

Reply:
reverted, in this demo, I was able to reproduce in my FF too 👍